packages/core/drizzle/meta/0008_snapshot.json

```diff
@@ -660,7 +660,7 @@
         "provider": {
           "name": "provider",
           "type": "provider",
-          "typeSchema": "public",
+          "typeSchema": "latitude",
```

Comment:
:\ I don't get why this happened

Reply:
The first time was added as public. https://github.com/latitude-dev/latitude-llm/commit/8602afddbebafb18559a38595d62f96c05018389#diff-218de992584a1201775b354fc2048c6fa4555dcab5196b1745d3f24c1e85375cR18

Reply:
And we all had from that point forward the same enum in latitude and public schemas

Reply:
sure but don't change the past, do a migration to change the enum schema